.github/dependabot.yml

```diff
@@ -31,10 +33,12 @@ updates:
 
   - package-ecosystem: "maven"
     directory: "/"
-    open-pull-requests-limit: 20
+    open-pull-requests-limit: 50
```

Comment:
Is there a reason this is 50 and the others 30?   Expectations of more changes here?

Reply:
This is covered in [a prior comment](https://github.com/eclipse/jetty.project/pull/8140#discussion_r892882429).